### ISSUE TITLE
Creating new app fails with MySql backend

### ISSUE THREAD

**Issue by viktortnk** [PredictionIO/PredictionIO#189]
Invoking `pio app new <name>` fails with following exception (Mysql 5.6/5.7):

```
[ERROR] [StatementExecutor$$anon$1] SQL execution failed (Reason: Invalid default value for 'creationTime'):

   create table if not exists pio_event_6 ( id varchar(32) not null primary key, event text not null, entityType text not null, entityId text not null, targetEntityType text, targetEntityId text, properties text, eventTime timestamp not null, eventTimeZone varchar(50) not null, tags text, prId text, creationTime timestamp not null, creationTimeZone varchar(50) not null)

Exception in thread "main" com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Invalid default value for 'creationTime'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.Util.getInstance(Util.java:372)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:980)
```

Can specifying default value be a potential solution?

http://stackoverflow.com/questions/168736/how-do-you-set-a-default-value-for-a-mysql-datetime-column

**Comment by EmergentOrder** [comment]
This can be addressed w/ MySQL config, see this group post https://groups.google.com/forum/#!searchin/predictionio-user/default$20sql/predictionio-user/S4DRjHsPoDU/TusHMJGaBAAJ .
Or as you said, specifying a default would also do it. 
Seems to affect only newer versions of MySQL than the current default

**Comment by viktortnk** [comment]
Thanks @EmergentOrder 

Changing MySql mode helped. Are there any long-term plans for addressing this issue?

I see `<column_name> timestamp DEFAULT CURRENT_TIMESTAMP` supported in PG as well. Should we try to apply this change globally?
I can probably help with testing it with different versions

**Comment by EmergentOrder** [comment]
I applied this changed globally, tested in PGSQL.
Will PR soon.

**Comment by EmergentOrder** [comment]
Fixed in PR #199

**Comment by EmergentOrder** [comment]
PR has been merged
